splitter/splitter.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/http;
+import ballerina/mime;
+import ballerina/url;
+
+type ReminderRequest record {
+    string date;
+    Event[] events;
+};
+
+type Event record {|
+    string eventName;
+    Attendee[] attendees;
+|};
+
+type Attendee record {|
+    string name;
+    string number;
+|};
+
+final string fromNo = "+15005550006";
+final string twilioSID = "VAC1829a53d52f41b4b2b1cc003c0026aa8";
+final string  apiVersion = "2010-04-01";
+
+final http:Client twilioClient = check new ("http://api.twilio.com.balmock.io");
+
+service /events on new http:Listener(8080) {
```

Comment:
For other service sample we used something like `/api/v1/...`. Shall we follow the same conventions here as well.